index.bs

```diff
@@ -596,30 +607,7 @@ Registry instances are initially created with the internal slots described in th
 1. Return the result of promise-calling _hook_(_name_, _referrer_).
 </emu-alg>
 
-<h4 id="fulfill-fetch" aoid="FulfillFetch">FulfillFetch(loader, entry, payload)</h4>
```

Comment:
finally we are getting rid of these methods, there is no more book keeping, everything happens naturally thru the pipeline.